README.md

```diff
@@ -1,25 +1,96 @@
 # AtlasTests
 
-To start you have to install cucumber plugin to IntelliJ Idea: https://plugins.jetbrains.com/plugin/7212-cucumber-for-java
-Please take attention on version of your Idea, to download correct version of plugin. (Help -> About)
+### Common and configuration
 
 Clone this project to your computer.
 
-Create file "application.properties" with parameters:
-login
-password
-downloadpath
+You have to set correct parameters in *.feature files (blue color), e.g. you have to set existing cohorts
```

Comment:
`You have`? Why not passive form? e.g. `It is required to set blah-blah-blah`

Reply:
The same as above